pages/k8s/charm-tigera-secure-ee.md

```diff
@@ -0,0 +1,109 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Tigera secure EE charm"
+  description: Kubernetes-master Charm reference
+keywords: kubernetes-master, charm, config
+tags: [reference]
+sidebar: k8smain-sidebar
+permalink: charm-tigera-secure-ee.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+# Tigera Secure EE Charm
```

Comment:
The charm-tigera-secure-ee page ([preview](https://deploy-preview-296--cdk-docs-next.netlify.com/kubernetes/docs/charm-tigera-secure-ee)) is showing 2 title headers. Maybe remove this line?